packages/grafana-data/src/types/datasource.ts

```diff
@@ -75,8 +75,8 @@ export class DataSourcePlugin<
     return this;
   }
 
-  setExploreStartPage(ExploreStartPage: ComponentType<ExploreStartPageProps>) {
-    this.components.ExploreStartPage = ExploreStartPage;
+  setQueryEditorHelp(QueryEditorHelp: ComponentType<QueryEditorHelpProps>) {
```

Comment:
would be good if we could find a runtime way to support this function still to not break old plugins like x-ray: https://github.com/grafana/x-ray-datasource/blob/master/src/module.ts#L13 

We could either deprecate it, or just a runtime way to bind this.setExploreStartPage = this.setQueryEditorHelp